### PR TITLE
Add Indonesia vehicle license (police) number

### PR DESCRIPTION
**Pull request by matriphe**
This method will produce something like: `B 1234 XYZ`, `AD 1234 VW`, `RI 1`, or `CD 53`

### CODE REVIEW

File: src/Faker/Provider/id_ID/Person.php [hunk 1]
```diff
@@ -244,13 +285,68 @@ public static function lastNameFemale()
...
+    }
+
+    /**
+     * Return vehicle license (police) number.
+     *
+     * @return string vehicle license number
+     */
+    public static function personalLicenseNumber()
+    {
+        $license = static::randomElement(static::$vehicleAreaCode);
```
Comment (fzaninotto) [inline src/Faker/Provider/id_ID/Person.php]: better: use `$this->generator->parse()` with a format string. See other providers as a usage example.

**Comment by fzaninotto** [comment]
Can you please rebase?

**Comment by matriphe** [comment]
Oops.. This branch had left so far behind. Close it for now.

**Comment by matriphe** [comment]
I think it would be better to go different provider, for example `Vehicle` provider.

**Pull request closed without merging**